tests/sql_mode/data/db1.prepare.sql

```diff
@@ -0,0 +1,42 @@
+set @@session.SQL_MODE='';
+
+drop database if exists `sql_mode`;
+create database `sql_mode`;
+use `sql_mode`;
+CREATE TABLE `t_1` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(60),
+  `num` int,
+  `dt` datetime,
+  PRIMARY KEY (id)
+);
+
+-- test sql_mode PIPES_AS_CONCAT
+insert into t_1(num) values('pipes'||'as'||'concat');
```

Comment:
why this can be inserted into an `int` field 😂

Reply:
Because it exists here as a bit operation OR.